README.md

```diff
@@ -385,25 +385,52 @@ To enable New Relic, simply bind a New Relic service to this app and settings wi
 
 ### AppDynamics
 
-To enable AppDynamics, configure the following environment variables:
+All the information collected with AppDynamics Java Agent and Machine Agent is pushed to Controller and displayed on the 
+AppDynamics WEB interface.
 
-```plaintext
-APPDYNAMICS_CONTROLLER_PORT
-APPDYNAMICS_CONTROLLER_SSL_ENABLED
-APPDYNAMICS_CONTROLLER_HOST_NAME
-APPDYNAMICS_AGENT_APPLICATION_NAME
-APPDYNAMICS_AGENT_ACCOUNT_NAME
-APPDYNAMICS_AGENT_ACCOUNT_ACCESS_KEY
-APPDYNAMICS_AGENT_NODE_NAME *
-APPDYNAMICS_AGENT_TIER_NAME
-```
+#### Basic functionality
 
-\* The `APPDYNAMICS_AGENT_NODE_NAME` environment variable will be appended with the value of the `CF_INSTANCE_ID` variable. If you use `my-app` for `APPDYNAMICS_AGENT_NODE_NAME` , the AppDynamics agent will be configured as `my-app-0` for instance `0` and `my-app-1` for instance `1` , etc.
-
-If you have any environment variable that starts with `APPDYNAMICS_` , the AppDynamics Java Agent will be configured for your application. At the moment only agent version `21.11.1.33280` is available. After configuring these environment variables, restage your app for the agent to be enabled.
+To enable AppDynamics, configure the following environment variables. 
+These settings enable basic (Java Agent related) AppDynamics features.
+More information here: [Environment variables](https://docs.appdynamics.com/22.2/en/application-monitoring/install-app-server-agents/java-agent/install-the-java-agent/use-environment-variables-for-java-agent-settings#UseEnvironmentVariablesforJavaAgentSettings-EnvironmentVariables).
 
 Please note that AppDynamics requires Mendix 6.2 or higher.
 
+| Environment Variable | Value Example | Description |
+|-|-|-|
+| `APPDYNAMICS_CONTROLLER_PORT` | 443 | Port of AppDynamics controller |
+| `APPDYNAMICS_CONTROLLER_SSL_ENABLED` | true | Set if SSL is required |
+| `APPDYNAMICS_CONTROLLER_HOST_NAME` | account-name.saas.appdynamics.com | Name of AppDynamics host without 'http://' |
+| `APPDYNAMICS_AGENT_APPLICATION_NAME` | myapp | Name of the app to be displayed on Controller UI |
+| `APPDYNAMICS_AGENT_ACCOUNT_NAME` | account-name | See 'License/Account' on the Controller UI |
+| `APPDYNAMICS_AGENT_ACCOUNT_ACCESS_KEY` | secret_key | See 'License/Account' on the Controller UI |
+| `APPDYNAMICS_AGENT_NODE_NAME` * | mynode | How a node is displayed on the Controller UI |
+| `APPDYNAMICS_AGENT_TIER_NAME` | mytier | How a tier is displayed on the Controller UI |
```

Comment:
This is good 💯  
Is there any documentation we can add here explaining Tiers and Nodes in detail?

Reply:
I added it below: https://github.com/mendix/cf-mendix-buildpack/pull/508/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R413